divide.py

```diff
@@ -4,3 +4,9 @@ def integer_division(dividend, divisor):
 
 def multiply_fun(a, b):
     return (a*b)
+
+
+A = 1
+B = 2
+C = multiply_fun(A, B)
```

Comment:
```suggestion
C = multiply_fun(A, B)
```
Can you name it differently, so we have the option to ad more variables to multiply_fun?